server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -73,12 +73,12 @@ public final DivTag render(ApplicantQuestionRendererParams params) {
                     .with(
                         TextFormatter.formatTextWithAriaLabel(
                             applicantQuestion.getQuestionHelpText(),
-                            /* preserveEmptyLines= */ true,
+                            /* preserveEmptyLines= */ false,
```

Comment:
Presumably this will have an effect on help text that contains empty lines in it? 

(For example if I have some help text like like:
"Please enter from the following options:

* foo
* bar"

will the empty line after "Please enter..." be removed?)

Not sure how much we care if so...

Reply:
The empty space is unnecessary because of the margin added on  mb-4. If there is no string then we don't want to render a line. We can add spacing with the margin.

Reply:
To clarify: I'm asking if an admin writes some help text which contains empty lines in the middle of their help text, will this remove them? Or does it only affect lines at the end/completely empty help text?

Reply:
Woops, accidentally clicked resolve, still getting the hang of this UI. 

This removes empty lines throughout the help text. Thanks for the catch! Keeping prserveEmptyLines to true would preserve the existing functionality and not interfere with the fix.